tests/node_tests.rs

```diff
@@ -1,18 +1,20 @@
-use dutch_auction::{Action, CreateConfig, Duration};
+use std::path::Path;
+
+use auction_io::io::*;
+use dutch_auction::WASM_BINARY_OPT;
 use gclient::{EventProcessor, GearApi, Result};
 use gear_lib::non_fungible_token::token::TokenMetadata;
 use gstd::prelude::*;
 use gstd::{ActorId, Encode};
 use nft_io::*;
 
-#[cfg(debug_assertions)]
-const PATH: &str = "../target/wasm32-unknown-unknown/debug/dutch_auction.opt.wasm";
-
-#[cfg(not(debug_assertions))]
-const PATH: &str = "../target/wasm32-unknown-unknown/release/dutch_auction.opt.wasm";
+const NFT_PATH: &str = "target/nft-0.2.5.opt.wasm";
 
 #[tokio::test]
-async fn buy() -> Result<()> {
+// #[ignore]
```

Comment:
```suggestion
#[ignore]
```
Tests with `gclient` should be ignored.